### PR TITLE
feat: Added example for nextjs app router

### PR DESCRIPTION
**Pull request by jagadeeshgade008**
**What's done**
Added app router example
created separate folders for page router and app router

### CODE REVIEW

**Comment by fahad19** [comment]
thanks for the PR, @jagadeeshgade008!

I think instead of moving the entire app to a different directory, what we could do is only add a new page that uses App Router. we don't have to delete everything else.

you can see how other examples, like Astro, are reusing the same SDK instance when rendering pages:

- instance: https://github.com/featurevisor/featurevisor-example-astro/blob/main/src/featurevisor.mjs
- usage in routes: https://github.com/featurevisor/featurevisor-example-astro/blob/main/src/pages/index.astro#L4

Above is just Astro's example, but I assume the implementation will be very similar with NextJS App Router as well.

We have a NuxtJS example too using Vue: https://github.com/featurevisor/featurevisor-example-nuxt

**Comment by jagadeeshgade008** [comment]
@fahad19,
I have updated app router example in same directory.

File: app/approuter/page.tsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import Image from "next/image";
```
Comment (fahad19) [inline app/approuter/page.tsx]: @jagadeeshgade008: is it possible to keep `app` directory inside `src`?

trying to find a similar example here: https://github.com/vercel/next.js/tree/canary/examples/blog-starter/src/app

(my knowledge with Next.js App Router is very limited)

that way, we could also keep the `featurevisor.ts` file inside `src`. so all code basically goes inside single `src` directory nicely.

Comment (jagadeeshgade008) [reply app/approuter/page.tsx]: @fahad19 I moved the app directory to the src directory

**Comment by jagadeeshgade008** [comment]
@fahad19 I took your suggestion and moved the app directory to the src directory. Thanks for the helpful tip!

**Review by fahad19 — APPROVED** [review]

**Comment by fahad19** [comment]
thanks so much, @jagadeeshgade008! merged 🚀

**Pull request merged**